### PR TITLE
Polishing

### PR DESCRIPTION
**Pull request by jbduncan**
- Rename variables from `snake_case` to `camelCase`.
- Remove unnecessary type information.
- Use `foreach` loops over traditional `for` loops where possible.
- Remove unnecessary explicit unboxing casts.
- Make unnecessarily public functions private.

### CODE REVIEW

File: jung-algorithms/src/test/java/edu/uci/ics/jung/algorithms/shortestpath/TestShortestPath.java [hunk 1]
```diff
@@ -479,35 +478,35 @@ public String get() {
 
   @Override
   protected void setUp() {
-    edgeWeights = new HashMap<Integer, Number>();
+    edgeWeights = new HashMap<>();
     nev = edgeWeights::get;
     dg = NetworkBuilder.directed().allowsParallelEdges(true).allowsSelfLoops(true).build();
-    for (int i = 0; i < dg_distances.length; i++) {
+    for (double[] dg_distance : dg_distances) {
```
Comment (jrtom) [inline jung-algorithms/src/test/java/edu/uci/ics/jung/algorithms/shortestpath/TestShortestPath.java]: I don't think that in this case (or the one below on line 492) that it's actually a good idea to make this a foreach loop.

This is not actually doing _anything_ with the `dg_distances` array; it's just saying "do [X] as many times as there are elements in the dg_distances array".  That is: we don't need a `dg_distance` variable, and so creating it seems a bit odd.

Comment (jbduncan) [reply jung-algorithms/src/test/java/edu/uci/ics/jung/algorithms/shortestpath/TestShortestPath.java]: Good point! I hadn't noticed that when I created this change. (It was suggested amongst many other potential changes in my IntelliJ when I ran a manual _"Inspect Code..."_, so it slipped my notice that foreach could've been a bad idea here.)

Reverting. :)

**Comment by jbduncan** [comment]
@jrtom I've pushed a new squashed commit for your (re)viewing pleasure. :)

(It occurs to me with hindsight that I should probably keep corrective commits separate until the PR is ready to be merged, as otherwise it's difficult to notice changes between corrections. I'll aim to improve in this area in the future.

In the meantime, know that the changes I've made so far are reverting lines 484 and 492 in `TestShortestPath.java` and adding a new TODO to `DijkstraShortestPath.java`.)

**Comment by jrtom** [comment]
This looks fine, but in the interim we've merged the common.graph branch into master and so that's where these changes should go.

**Comment by tomnelson** [comment]
I was going to ask you if you would be deleting common.graph now, or
keeping it as a development Branch for development PRs

On Nov 22, 2017 1:25 PM, "jrtom" <notifications@github.com> wrote:

This looks fine, but in the interim we've merged the common.graph branch
into master and so that's where these changes should go.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/jrtom/jung/pull/144#issuecomment-346435117>, or mute
the thread

**Comment by jrtom** [comment]
@tomnelson If we start doing more speculative work that we're not sure we want to go into master, then we should create a custom branch for it.  I haven't deleted the common.graph branch yet primarily because of changes like this, that I didn't want to orphan (I don't know how Git handles such cases).

**Comment by jbduncan** [comment]
@jrtom I've changed the target branch from `common.graph` to `master` as you requested. Please let me know if I should do anything else before merging!

**Pull request merged**